packages/starlight-typedoc/tests/unit/typedoc.test.ts

```diff
@@ -91,3 +91,69 @@ test('should not add `README.md` module files for multiple entry points', async
   expect(writeFileSyncSpy).toHaveBeenCalled()
   expect(filePaths.some((filePath) => filePath.endsWith('README.md'))).toBe(false)
 })
+
+test('should support overriding typedoc-plugin-markdown readme and index page generation', async () => {
+  await generateTypeDoc({
+    ...starlightTypeDocOptions,
+    typeDoc: {
+      readme: 'README.md',
```

Comment:
```suggestion
      ...starlightTypeDocOptions.typeDoc,
      readme: 'README.md',
```

Missed that in the previous PR but it avoid spamming the console during tests with useless logs. Might do a helper later to avoid passing that in explicitly but I'll wait to see if I want to refactor the tests at some point.